build.java

```diff
@@ -807,19 +945,19 @@ private static boolean notMavenOrg(String line)
     private static Function<String, String> prependMavenProxy(Options options)
     {
         return line ->
-        {
-            var mavenBaseURL = String.format("\"%s/\"", options.mavenProxy);
-            return line.contains("_mavenRepoBaseURLs")
-                ? line.replaceFirst("\\[", String.format("[ %s", mavenBaseURL))
+            line.contains("_mavenRepoBaseURLs")
+                ? line.replaceFirst("\\[", String.format("[ %s", String.format("\"%s/\"", options.mavenProxy)))
```

Comment:
```suggestion
                ? line.replaceFirst("\\[", String.format("[ \"%s/\"", options.mavenProxy))
```
Isn't this essentially the same?

Reply:
It is.   Formatting...  Will revert if you find it better.

Reply:
I think it's "simpler" and more "readable". That's my opinion though... feel free to keep either version :)

Reply:
I just noticed that we might not be talking about the same thing here.
When I asked if this is essentially the same I was talking about my suggestion in https://github.com/graalvm/mandrel-packaging/pull/71#discussion_r516879370 which inlines `String.format("\"%s/\"", options.mavenProxy)`.

Your initial change that inlines `mavenBaseURL` is totally fine!